canonical-web-and-design/canonicalwebteam.blog
# Adds author endpoint

Adds author endpoint
Fixes https://github.com/canonical-web-and-design/base-squad/issues/525

## QA
- Use https://github.com/canonical-web-and-design/www.ubuntu.com and replace `canonicalwebteam.blog==1.5.X` with `git+https://github.com/b-m-f/blog-extension@add-author-pages#egg=canonicalwebteam.blog` in `requirements.txt`
- `./run` the site
- create a template at `blog/author.html` and fill it with `{{author}}{{latest_articles}}`
- head to `localhost;8001/blog/author/b-m-f` and verify that the necessary author information and the last (max.5) articles are displayed (2 for this user).
